index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Capstone</title>
+  <link rel="stylesheet" href="./style.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css" integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700&display=swap" rel="stylesheet">
+</head>
+<body>
+  <div id="menu-section" class="menu-section">
+      <button id="menu-toggle" class="menu-toggle"><i class="fa-solid fa-xmark"></i></button>
+      <ul class="menu-list">
+          <li class="menu-item"><a href="#about">About</a></li>
+          <li class="menu-item"><a href="#program">Program</a></li>
+          <li class="menu-item"><a href="#Join">Join</a></li>
+          <li class="menu-item"><a href="#sponsor">Sponsor</a></li>
+          <li class="menu-item"><a href="#news">News</a></li>
```

Comment:
- [ ] Kindly add 'Home' to your navbar and mobile menu since when the user on the About page can't access the home from the navigation panel.